packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/snapshot_control_pane.dart

```diff
@@ -46,18 +46,29 @@ class SnapshotControlPane extends StatelessWidget {
                 ],
               ],
             ),
-            ToolbarAction(
-              icon: Icons.clear,
-              tooltip: 'Delete snapshot',
-              onPressed: isProcessing
-                  ? null
-                  : () {
-                      controller.deleteCurrentSnapshot();
-                      ga.select(
-                        gac.memory,
-                        gac.MemoryEvent.diffSnapshotDelete,
-                      );
-                    },
+            Row(
+              children: [
+                ToolbarAction(
+                  icon: Icons.clear,
+                  tooltip: 'Delete snapshot',
+                  onPressed: isProcessing
+                      ? null
+                      : () {
+                          controller.deleteCurrentSnapshot();
+                          ga.select(
+                            gac.memory,
+                            gac.MemoryEvent.diffSnapshotDelete,
+                          );
+                        },
+                ),
+                ToolbarAction(
```

Comment:
Let's put this action on the left side of these controls instead. Next to the CSV dropdown. This would match what is on the other memory tabs.